### PR TITLE
Fix issues in xplat project

### PR DESCRIPTION
**Pull request by mfilippov**
Fix some issues in xplat branch.

### CODE REVIEW

File: src/.nuget/NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (rainersigwald) [inline src/.nuget/NuGet.Config]: This file was intentionally deleted in 1ea71faaa6d71d88e37b3f98ff93f379b8707636. I think it'd be better to update the solution item to point to the new file in the root of the repo.

File: src/xunit.runner.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (rainersigwald) [inline src/xunit.runner.json]: This file moved to `targets/xunit.runner.json`--can you update the `.sln` to point to that location?

**Comment by jeffkl** [comment]
@dotnet-bot test OSX Build for CoreCLR please

File: src/xunit.runner.json (hunk 2 above)
Comment (mfilippov) [inline src/xunit.runner.json]: No problem I do it.

**Comment by mfilippov** [comment]
@rainersigwald I fix sln file and force push.

**Comment by rainersigwald** [comment]
Thanks!

**Pull request merged**